src/wash/particle.cpp

```diff
@@ -0,0 +1,16 @@
+#include "wash.hpp"
+
+namespace wash {
+    void Particle::initialise_particle_forces() {
```

Comment:
Looks like I forgot to copy this function from the old API, thanks for fixing this